proto/cosmos/auth/v1beta1/query.proto

```diff
@@ -176,3 +185,32 @@ message QueryAccountAddressByIDRequest {
 message QueryAccountAddressByIDResponse {
   string account_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
 }
+
+// QueryAccountInfoRequest is the Query/AccountInfo request type.
+message QueryAccountInfoRequest {
+
+  // address is the account address string.
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+// QueryAccountInfoResponse is the Query/AccountInfo response type.
+message QueryAccountInfoResponse {
+
+  // info is the account info.
+  AccountInfo info = 1;
+}
+
+// AccountInfo describes account info common to all account types.
+message AccountInfo {
```

Comment:
I am probably missing some context here, but why is this so similar to `cosmos/auth/v1beta1/auth.proto/BaseAccount`? why have 2 very similar message types in Auth?

Reply:
Right, given we've moved this to `v1beta1` we can use `BaseAccount`. We can discuss the v1 ideas more in #13211